docs/operate/customize/require_community.md

```diff
@@ -0,0 +1,114 @@
+# Require Community for Record Publication
+
+## Overview
```

Comment:
minor: after going through countless doc pages, I've found that removing the "Overview" header and just diving into the text directly flows better and declutters the page and table of content.